web/src/message_store.ts

```diff
@@ -172,23 +173,23 @@ export function get(message_id: number): Message | undefined {
 
 export function get_pm_emails(message: Message): string {
     const user_ids = people.pm_with_user_ids(message) ?? [];
-    const emails = user_ids
-        .map((user_id) => {
-            const person = people.maybe_get_user_by_id(user_id);
-            if (!person) {
-                blueslip.error("Unknown user id", {user_id});
-                return "?";
-            }
-            return person.email;
-        })
-        .sort();
-
-    return emails.join(", ");
+    const emails = user_ids.map((user_id) => {
+        const person = people.maybe_get_user_by_id(user_id);
+        if (!person) {
+            blueslip.error("Unknown user id", {user_id});
+            return "?";
+        }
+        return person.email;
+    });
+
+    return people.sort_emails(emails, user_ids).join(", ");
```

Comment:
Thank you for pointing it out. I checked it and you are right. It is not used anywhere where we display names.
I think I sorted here because it was a similar function as other two ( this also gets emails from id and sort them ). But we don't need this for displaying names so we can remove it.